NSURLSessionCancellation/ViewController.swift

```diff
@@ -11,95 +11,95 @@ import UIKit
 class ViewController: UIViewController {
 
     // A sample URL that takes a while to download (so that cancellaton can be demonstrated)
-    
+
     fileprivate let url = URL(string: "http://yahoo.com")!
-    
+
     /// A default URL session
-    
+
     fileprivate let session = URLSession(configuration: URLSessionConfiguration.default)
 
-    
+
     // MARK: - UI Actions
-    
+
     @IBAction func downloadButtonPressed(_ sender: AnyObject) {
-        
-        self.setLoadingSpinnerVisible(true)
-        
-        // Download data 
-        
-        self.session.dataTask(with: self.url, completionHandler: { (data, response, error) in
-            
-            DispatchQueue.main.async(execute: { 
-             
+
+        setLoadingSpinnerVisible(true)
+
+        // Download data
+
+        session.dataTask(with: url, completionHandler: { (data, response, error) in
```

Comment:
Ah that's a good point because in RW style guide this is what they said:

> Use trailing closure syntax only if there's a single closure expression parameter at the end of the argument list.

And I forgot to update the code. Thanks